torchaudio_contrib/functional.py

```diff
@@ -46,60 +46,51 @@ def _hertz_to_mel(hz, htk):
                        torch.log(hz / min_log_hz) / logstep, mel)
 
 
-def stft(waveforms, fft_len, hop_len, window,
-         pad=0, pad_mode="reflect", **kwargs):
+def stft(waveforms, fft_len, hop_len=None, win_len=None, window=None,
+         center=True, pad_mode='reflect', normalized=False, onesided=True):
```

Comment:
If we're just wrapping `torch.stft` to allow for multichannel, do we have to explicitly pass the parameters? Can we have something like:
```python
def stft(waveforms, **stft_kwargs):
    ... # mutli-channel
    complex_specgrams = torch.stft(waveforms, **stft_kwargs)
    return complex_specgrams
```
Then if there were any changes to `torch.stft` we wouldn't be explicitly out of sync...?

Reply:
Sounds good. I wanted to make things more explicit, but probably a little more verbose docstring would do that.